supernode/daemon/util/filter.go

```diff
@@ -28,19 +28,19 @@ import (
 )
 
 const (
-	// PAGENUM identity the page number of the data.
+	// PAGENUM identities the page number of the data.
```

Comment:
I think `identifies` is much appropriate

Reply:
As you can see, I have already modified it.
But I can't understand what you mean,“I think identifies is much appropriate.”

Reply:
identity is a verb, but identity is a noun.

Reply:
Wow, you are great.
Thank you.